configs/foreman/nightly.yaml

```diff
@@ -570,6 +567,7 @@
       srpm-build:
         - bash
         - centos-release
+        - redhat-release-server
```

Comment:
while I find seeing rh-rel-server weird in this list, here centos is not removed?

Reply:
Yea, its a good callout. I'm just the messenger, don't throw :egg: :egg: :egg:

Reply:
Easter is over anyways :rabbit: